db/migrate/20181028092134_add_user_to_password_2.rb

```diff
@@ -0,0 +1,11 @@
+class AddUserToPassword2 < ActiveRecord::Migration
+  def self.up
+    add_column :passwords, :user_id, :integer
```

Comment:
I did have devise previously and it seems I didn't not properly remove it.  The passwords table already has `user_id`.  We should do this instead to be safe and avoid breaking the migration.
```suggestion
    unless column_exists? : passwords, : user_id
        add_column :passwords, :user_id, :integer
        add_index :passwords, :user_id
    end
```